shortcodes/pmpro_account.php

```diff
@@ -142,7 +142,7 @@ function pmpro_shortcode_account($atts, $content=null, $code="")
 												$expiration_text .= ' ' . date_i18n( get_option( 'time_format', __( 'g:i a' ) ), $level->enddate );
 											}
 										} else {
-											$expiration_text .= esc_html_e( '&#8212;', 'paid-memberships-pro' );
+											$expiration_text .= '&#8212;';

```

Comment:
Let's still localize this though, with context.
```suggestion
											$expiration_text .= esc_html_x( '&#8212;', 'A dash is shown when there is no expiration date.', 'paid-memberships-pro' );
```